x-pack/plugins/security_solution/scripts/run_cypress/get_ftr_config.ts

```diff
@@ -171,17 +171,6 @@ export const getFTRConfig = ({
         }
       }
 
-      // Serverless Specific
-      if (vars.serverless) {
-        log.info(`Serverless mode detected`);
-
-        vars.esTestCluster.serverArgs.push(
-          `xpack.security.authc.realms.saml.cloud-saml-kibana.sp.entity_id=http://host.docker.internal:${kibanaPort}`,
-          `xpack.security.authc.realms.saml.cloud-saml-kibana.sp.logout=http://host.docker.internal:${kibanaPort}/logout`,
-          `xpack.security.authc.realms.saml.cloud-saml-kibana.sp.acs=http://host.docker.internal:${kibanaPort}/api/security/saml/callback`
-        );
-      }
```

Comment:
It was the reason for Cypress tests to fail. Please avoid overriding SAML configuration for both Kibana & ES, we keep things up-to-date in [x-pack/test_serverless/shared/config.base.ts](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/shared/config.base.ts)

Reply:
@dmlemeshko if serverless ES is running in docker and Kibana is running locally, then https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/shared/config.base.ts#L78-L80
`http://localhost:${servers.kibana.port}` is not going to be reachable from ES docker container, because `localhost` will be referencing the container itself, not the host where Kibana is running

Reply:
> http://localhost:${servers.kibana.port} is not going to be reachable from ES docker container, because localhost will be referencing the container itself, not the host where Kibana is running

Just to confirm my understanding, why do you require Kibana to be accessible from the ES container? If you're referring to URLs in `sp.{logout|acs}`, then it doesn't matter — ES solely uses these URLs to generate a special URL for redirecting users during login or logout. Hence, the important part here is that the user's browser can access these URLs.